pkgs/applications/display-managers/lightdm-slick-greeter/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, vala
+, intltool
+, autoreconfHook
+, wrapGAppsHook
+, lightdm
+, gtk3
+, pixman
+, libcanberra
+, libX11
+, libXext
+, linkFarm
+, lightdm_slick_greeter
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lightdm-slick-greeter";
+  version = "1.5.9";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = "slick-greeter";
+    rev = "f1527015df9d0315c0a2344de0a91bea1559484e";
+    sha256 = "sha256-UEzidH4ZWggcOWHHuAclHbbgATDBdogL99Ze0PlwRoc=";
+  };
+
+  nativeBuildInputs = [ pkg-config vala intltool autoreconfHook wrapGAppsHook ];
+  buildInputs = [ lightdm gtk3 pixman libcanberra libX11 libXext ];
+
+  postPatch = ''
+    mkdir m4
```

Comment:
Could you add a comment explaining why this is done?

Reply:
Got it; the configure was failing during the autoreconfPhase.